ePanchayat.DB/StoredProcedures/HouseGet_sp.sql

```diff
@@ -0,0 +1,30 @@
+USE [ePanchayat]
+GO
+
+IF OBJECT_ID('dbo.HouseGet_sp') IS NOT NULL
+BEGIN
+	DROP PROCEDURE dbo.HouseGet_sp
+END
+GO
+
+CREATE PROCEDURE dbo.HouseGet_sp
+(
+	@HouseId INT = NULL
+)
+AS
+BEGIN
+	SELECT
+		VehicleId
+		,Category
+		,RegistrationNumber
+		,OwnerId
+		,LastModifiedOn
+		,LastModifiedBy
+		,IsActive
+	FROM
+		dbo.House_tbl

```

Comment:
Let we perform join with user_tbl to get user details also for Owner and ModifiedBy user.